Cargo.toml

```
@@ -13,17 +13,17 @@ description = "Atomic types of the FuelVM."
 [dependencies]
 rand = { version = "0.8", default-features = false, optional = true }
 serde = { version = "1.0", default-features = false, features = ["derive"], optional = true }
-serde-big-array = { version = "0.3.2", default-features = false, optional = true }
 
 [dev-dependencies]
 bincode = "1.3"
 fuel-types = { path = ".", features = ["random"] }
 hex = "0.4"
 rand = { version = "0.8", default-features = false, features = ["std_rng"] }
+serde_json = "1.0"
 
 [features]
 default = ["std", "serde?/default"]
 alloc = ["serde?/alloc"]
 random = ["rand"]
-serde = ["dep:serde", "serde-big-array"]
+serde = ["dep:serde", "alloc"]
```

alloc is no longer a prereq of serde now

```suggestion
serde = ["dep:serde"]
```